### ISSUE TITLE
Could not load file or assembly 'System.Configuration.ConfigurationManager'. The system cannot find the file specified.

### ISSUE THREAD

**Issue by ghost** [Azure/azure-functions-host#7777]
### Description

Running a function app on consumption plan and since today the functions won't run (respond with 500 internal server error). The error message on the logs is the following:

```
Could not load file or assembly 'System.Configuration.ConfigurationManager, Version=4.0.3.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.
```

The System.Configuration package is not used by the function.

### Configuration

- Timestamp: 2021-10-15T13:18:21.634
- Function App version: ~4
- Invocation ID: 47f64639-0b64-41aa-8bea-afaa6fe48684
- Region: Norway East

Expected: Http status 200
Actual: Http status 500 with error message

### Other information
F# programming language and .NET version 6

**Comment by ghost** [comment]
Issue resolved by adding the following:
```
<PropertyGroup>
  <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
</PropertyGroup>
```

**Comment by cjaliaga** [comment]
@konstantpapp could it be possible for you to share your fsproj file?

**Comment by ghost** [comment]
Sure @cjaliaga, that's how the latest working version looks like:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
    <TargetFramework>net6.0</TargetFramework>
  </PropertyGroup>
  <PropertyGroup>
    <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.0-beta.5" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage.Blobs" Version="5.0.0-beta.5" />
    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.0.0-preview2" />
    <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.12" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.CosmosDB" Version="4.0.0-preview1" />
    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.21.0" />
    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
    <PackageReference Include="System.IO" Version="4.3.0" />
  </ItemGroup>
  <ItemGroup>
    <Compile Include="User.fs" />
  </ItemGroup>
  <ItemGroup>
    <Content Include="host.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
    <Content Include="local.settings.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
    </Content>
  </ItemGroup>
</Project>
```

**Comment by cjaliaga** [comment]
Thanks @konstantpapp.

A new version of `Microsoft.NET.Sdk.Functions` has been published, 4.0.0: https://www.nuget.org/packages/Microsoft.NET.Sdk.Functions/4.0.0

```xml
<PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.0.0" />
````

If you update to this one the workaround should not be necessary anymore and could be removed:

```xml
<PropertyGroup>
    <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
</PropertyGroup>
```

Please let us know if this doesn't work for you.

**Comment by ghost** [comment]
Confirming that the above work, thank you @cjaliaga.
